das/read_limited.go

```diff
@@ -36,7 +36,7 @@ func (s *readLimitedStorageService) Close(ctx context.Context) error {
 	return nil
 }
 
-func (lss *readLimitedStorageService) ExpirationPolicy(ctx context.Context) ExpirationPolicy {
+func (lss *readLimitedStorageService) ExpirationPolicy(ctx context.Context) arbstate.ExpirationPolicy {
 	return -1
```

Comment:
This should return some other value, not an invalid one.  Perhaps this and the empty storage service should return a new `DiscardImmediately` strategy.

Reply:
Actually this was added by mistake, we do not need to implement it since will inherit the function from the reader arbstate.DataAvailabilityReader provided in NewReadLimitedStorageService.